src/api/components/components.ts

```diff
@@ -3703,16 +3752,17 @@ class ValueRandomMechanic extends ProMechanic {
 			data:         [
 				new StringSelect('Key', 'key', 'value')
 					.setTooltip('The unique key to store the value under. This key can be used in place of attribute values to use the stored value'),
-				new BooleanSelect('Integer', 'integer', false)
```

Comment:
It's there a reason this option was removed?

Reply:
Maybe I forgot it when I merged the branch